_studio/shared/include/libmfx_core_hw.h

```diff
@@ -18,15 +18,54 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-#include "mfx_common.h"
-
-
 #ifndef __LIBMFX_CORE__HW_H__
 #define __LIBMFX_CORE__HW_H__
 
+#include "mfx_common.h"
 #include "umc_va_base.h"
+#include <memory>
+#include <mutex>
+#include <vector>
+
+struct PciDisplayDevice
+{
+    mfxU32 vendor_id;
```

Comment:
looks like a good place for default initialization with invalid vendor & device id